segmentstore/server/src/main/java/io/pravega/segmentstore/server/CachePolicy.java

```diff
@@ -11,71 +11,101 @@
 
 import com.google.common.base.Preconditions;
 import java.time.Duration;
+import lombok.Getter;
 
 /**
  * Represents a Policy for a CacheManager.
  */
 public class CachePolicy {
     //region Members
-    public static final CachePolicy INFINITE = new CachePolicy(Long.MAX_VALUE, Duration.ofSeconds(Integer.MAX_VALUE), Duration.ofSeconds(Integer.MAX_VALUE));
+    public static final CachePolicy INFINITE = new CachePolicy(Long.MAX_VALUE, 1.0, 1.0, Duration.ofSeconds(Integer.MAX_VALUE), Duration.ofSeconds(Integer.MAX_VALUE));
+    /**
+     * Unless specified via the constructor, defines the default value for {@link #getMaxUtilization()}.
+     */
+    public static final double DEFAULT_TARGET_UTILIZATION = 0.85;
+    /**
+     * Unless specified via the constructor, defines the default value for {@link #getMaxUtilization()}.
+     */
+    public static final double DEFAULT_MAX_UTILIZATION = 0.98;
```

Comment:
When it reaches max utilization, we will apply max throttling. That means that every incoming request has a 25 second delay. If it doesn't drop below that, it will continue to throttle and not let anything in. If it drops just slightly, it the most it will let through is 1MB per container (BK write size). Even if we have 30 containers on a pod, the most that will go through is 30MB, which is 1% of the total cache size, at which point the throttler will again block until the cache drops below. However 30 containers with 3GB cache is a bad combination. More likely it should be 4-6 containers for every 3GB (we also need to deploy Pravega with sensible configuration values, as not all combinations will provide good outcomes).

Furthermore, you need to see how we get to 98%. At 90% we begin throttling so every increment from there we will apply more and more delays (pressure) and load the cache slower and slower. By the time we reach 98% it will be so slow that barely anything will get through. It's important to understand that 98% is not a hard, solid wall; we will gradually get there by climbing many stairs beginning at 90%.

In addition to that, cache management will evict unpinned entries when the capacity reaches 85%, so the throttling itself is a last resort measure.